quantum/pointing_device/pointing_device_auto_mouse.c

```diff
@@ -32,6 +33,9 @@ static auto_mouse_context_t auto_mouse_context = {
     .config.layer    = (uint8_t)(AUTO_MOUSE_DEFAULT_LAYER),
     .config.timeout  = (uint16_t)(AUTO_MOUSE_TIME),
     .config.debounce = (uint8_t)(AUTO_MOUSE_DEBOUNCE),
+#ifdef AUTO_MOUSE_ONESHOT
```

Comment:
To keep things more consistent with the nameng.
```suggestion
#ifdef POINTING_DEVICE_AUTO_MOUSE_ONESHOT
```